docs/samples/python/transformers_recognizer/transformer_recognizer.md

```diff
@@ -0,0 +1,43 @@
+# Run Presidio With Transformers Models
+
+This example demonstrates how to extract PII entities using transformers models.
+When initializing the `TransformersRecognizer`, choose from the following options:
+1. A string referencing an uploaded model to HuggingFace. Use this url to access all TokenClassification models - https://huggingface.co/models?pipeline_tag=token-classification&sort=downloads
+
+2. Initialize your own TokenClassificationPipeline instance using your custom transformers model and use it for inference.
+
+3. Provide the path to your own local custom trained model.
+
+!!! note "Note"
```

Comment:
I think the note is not rendered as expected

Reply:
@erann1987 this is how we render notes for the MKDocs website, e.g.:
![image](https://user-images.githubusercontent.com/3776619/208299026-53130fca-7667-4ba7-ae9a-9b222f336284.png)